library/src/main/java/com/getstream/sdk/chat/view/messageinput/MessageInputView.kt

```diff
@@ -321,14 +320,16 @@ class MessageInputView(context: Context, attrs: AttributeSet?) : RelativeLayout(
                 return false
             }
         }
-        if (inputContentInfo.linkUri == null) return false
-        val url = inputContentInfo.linkUri.toString()
-        val attachment = Attachment()
-        attachment.thumbUrl = url
-        attachment.titleLink = url
-        attachment.title = inputContentInfo.description.label.toString()
-        attachment.type = ModelType.attach_giphy
-        messageInputController.setSelectedAttachments(setOf(AttachmentMetaData(attachment)))
+        messageInputController.setSelectedAttachments(
+            setOf(
+                AttachmentMetaData(
+                    inputContentInfo.contentUri,
```

Comment:
Nit: For data classes, it's good to explicitly show args names in constructors. Someone can easily break this code if one adds new arguments to the constructor.